requirements.txt

```diff
@@ -1,14 +1,16 @@
 astroid==2.2.5
-Django==2.1
+Django==2.2
 django-cors-middleware==1.2.0
-django-extensions==1.7.1
-djangorestframework==3.4.4
+django-extensions==2.1.6
+djangorestframework==3.9.2
 isort==4.3.16
 lazy-object-proxy==1.3.1
 mccabe==0.6.1
+pycodestyle==2.5.0
```

Comment:
We already have pylint. We do not need pycodestyle.

Reply:
@IssaIan i have made the changes